website/docs/docs/dbt-versions/release-notes/74-Dec-2023/dec-sl-updates.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Updates and fixes: dbt Semantic Layer and MetricFlow updates for December 2023."
+description: "December 2023: Enhanced Tableau integration, BIGINT support, LookML to MetricFlow conversion, and deprecation of legacy features."
+sidebar_label: "Update ad fixes: dbt Semantic Layer and MetricFlow."
+sidebar_position: 08
+date: 2023-12-22
+---
+The dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability for the dbt Semantic Layer and MetricFlow. Here are the updates and fixes for December 2023.
+
+## Bug fixes
+- The dbt Semantic Layer integration with Tableau now supports queries that resolve to a "NOT IN" clause (for example: using "exclude" in the filtering user interface). Previously it wasn’t supported.
```

Comment:
do we need to give more info on the memory leak?

Reply:
I don't think so.... the errors that customers might have experienced from this could be different. We could maybe include a time period of when people might have been affected, but this could be hard to find.